### PR TITLE
Add simpleui v0.1.1

### LINKED ISSUE #7194: [Request] Add simpleui
**Library name:** simpleui
**Git repository url:** https://github.com/fengyuanchen/simpleui
**npm package url(optional):** simpleui
**Official homepage:** https://fengyuanchen.github.io/simpleui/

### PR DESCRIPTION
**Pull request by x09326**
PR for #7311 
@PeterDaveHello please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/fengyuanchen/simpleui
 * Official website (optional, not the repository): https://fengyuanchen.github.io/simpleui/
 * NPM package url (optional): simpleui
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 33
   - Count of watchers: 2
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 29
   - Downloads in the last month: 55

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with npm auto-update config added, close #7194 cc @fengyuanchen

### CODE REVIEW

File: ajax/libs/simpleui/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "name": "simpleui",
+  "filename": "simple.min.css",
+  "description": "A simple UI framework for building simple web projects.",
+  "version": "0.1.1",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fengyuanchen/simpleui.git"
+  },
+  "homepage": "http://fengyuanchen.github.io/simpleui",
```
Comment (PeterDaveHello) [inline ajax/libs/simpleui/package.json]: Please use https instead of http here.

Comment (x09326) [inline ajax/libs/simpleui/package.json]: I have already fixed it. @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Please squash/fixup your commits into only one commit, and rebase your branch with our latest master branch, this branch is 202 commits behind cdnjs:master.

**Comment by x09326** [comment]
I have already fixed up my commit, and rebased my branch. 
@PeterDaveHello Please help me check it.

**Comment by PeterDaveHello** [comment]
Looks you didn't rebase with our latest master branch, please discuss with other @cdnjs/intern2 members.

temp

**Comment by x09326** [comment]
Because I can't rebase, so I use cherry-pick.
@PeterDaveHello Please help me check it, thank you.

**Comment by PeterDaveHello** [comment]
@x09326 let @kennynaoh help you first

**Comment by x09326** [comment]
I have already rebased.
@PeterDaveHello Please help me review it, thank you.

**Comment by PeterDaveHello** [comment]
@x09326 great!

**Comment by fengyuanchen** [comment]
@PeterDaveHello and @x09326 Thank you both for adding Simple UI.

**Comment by PeterDaveHello** [comment]
:+1:

**Pull request merged**